### DISCUSSION TITLE
[QUESTION]:一个机器有多个服务的日志文件需要采集的时候 如何标记每个日志文件  目前file_log 配置fields字段无效

### DISCUSSION THREAD

**Discussion by suifengpiaosan** [alibaba/ilogtail#237] [Help] [answered]
目前的配置文件
enable: true
inputs:
  - Type: file_log
    LogPath: xxx
    FilePattern:xxx
    Fields:
      service: A
  - Type: file_log
    LogPath: xxx
    FilePattern:xxx
    Fields:
      service: B
processors:
  - Type: processor_add_fields
    Fields:
      aaa2: value3
      aaa3: value4
      aaa1: value1
flushers:
  - Type: flusher_kafka
    Brokers:
      - 10.190.50.204:9092
    Topic: access-log

**Comment by suifengpiaosan** [comment]
或者有没有其他的实践方式

**Comment by panawala** [comment, marked as answer]
file_log插件是支持通配符匹配文件路径，采集上来的日志会有一个标签字段`__tag__:__path__`表示文件的原始路径，
具体可以参考：[file_log插件用法](https://ilogtail.gitbook.io/ilogtail-docs/data-pipeline/input/file-log)
